pkg/reconciler/v1alpha1/route/resources/cluster_ingress.go

```diff
@@ -112,8 +109,9 @@ func makeClusterIngressRule(domains []string, ns string, targets traffic.Revisio
 		splits = append(splits, v1alpha1.ClusterIngressBackendSplit{
 			ClusterIngressBackend: v1alpha1.ClusterIngressBackend{
 				ServiceNamespace: ns,
-				ServiceName:      reconciler.GetServingK8SServiceNameForObj(t.TrafficTarget.RevisionName),
-				ServicePort:      intstr.FromInt(int(revisionresources.ServicePort)),
+				//				ServiceName:      reconciler.GetServingK8SServiceNameForObj(t.TrafficTarget.RevisionName),
```

Comment:
delete?

Reply:
This is going to need rewriting with my change anyways, so may as well do that as part of the merge.

Reply:
deleted.